src/audio/module_adapter/module/volume/volume.c

```diff
@@ -231,10 +231,8 @@ static const struct comp_zc_func_map zc_func_map[] = {
  * \param[in] ramp_time Time spent since ramp start as milliseconds Q29.3
  * \param[in] channel Current channel to update
  */
-static int32_t volume_linear_ramp(struct processing_module *mod, int32_t ramp_time, int channel)
+static inline int32_t volume_linear_ramp(struct vol_data *cd, int32_t ramp_time, int channel)
```

Comment:
I have a comment here, seems we always use cd to indicate data, however, for copier, use cd make sense, how about here to use vd to indicate it is volume data?